src/common/users/users.controller.ts

```diff
@@ -1,15 +1,15 @@
 import { Controller, Get, Version } from "@nestjs/common";
 
-@Controller({ path: "users" })
+@Controller()
 export class UsersController {
   @Version("1")
-  @Get()
+  @Get("users")
```

Comment:
both example seem to be in route level, use versioning at controller level for this one